arb.h

```diff
@@ -229,6 +229,12 @@ arb_set_ui(arb_t x, ulong y)
 }
 
 ARB_INLINE void
+arb_set_d(arb_t x, double y)
+{
+    arf_set_d(arb_midref(x), y);
+}
+
+ARB_INLINE void
```

Comment:
Wish I could make this line white without adding an extra new line character.  Should I just add the function to the end of the file so that this doesn't happen?